presto-main/src/test/java/com/facebook/presto/operator/TestFileFragmentResultCacheManager.java

```diff
@@ -51,11 +53,16 @@
 {
     private static final String SERIALIZED_PLAN_FRAGMENT_1 = "test plan fragment 1";
     private static final String SERIALIZED_PLAN_FRAGMENT_2 = "test plan fragment 2";
-    private static final Split SPLIT_1 = new Split(new ConnectorId("test"), new ConnectorTransactionHandle() {}, new TestingSplit(1));
-    private static final Split SPLIT_2 = new Split(new ConnectorId("test"), new ConnectorTransactionHandle() {}, new TestingSplit(2));
+    private static final Split SPLIT_1 = new Split(new ConnectorId("test"), new ConnectorTransactionHandle()
+    {
```

Comment:
Maybe revert the changes here and for `SPLIT_2` if they are unintended.

Reply:
fix